pvacseq/lib/call_iedb.py

```diff
@@ -46,12 +52,12 @@ def main(args_input = sys.argv[1:]):
     url = prediction_class_object.url
 
     response = requests.post(url, data=data)
-    if response.status_code == 500:
-        #Retry once
+    retries = 0
+    while response.status_code == 500 and retries < args.iedb_retries:
         response = requests.post(url, data=data)
-        if response.status_code == 500:
-            #Retry a second time
-            response = requests.post(url, data=data)
+        print("Retries %s" % retries)
```

Comment:
If we introduce a delay here, how long do you think it should be? I'm thinking 2 or 5 seconds but I have no idea what would really be reasonable.